tests/integration/test_yarn.py

```diff
@@ -74,6 +74,19 @@
             ),
             id="yarn_incorrect_checksum",
         ),
+        pytest.param(
+            utils.TestParameters(
+                repo="https://github.com/cachito-testing/cachi2-yarn-berry.git",
+                ref="28e1d12e19929e53997550451fc5598e56a162f7",
+                packages=({"path": ".", "type": "yarn"},),
+                check_output=False,
+                check_deps_checksums=False,
+                check_vendor_checksums=False,
+                expected_exit_code=2,
+                expected_output=" Yarn lockfile 'yarn.lock' missing, refusing to continue",
```

Comment:
You have an extra white space at the beginning of the message. Looking at the message though, we could probably benefit from making the integration test a tiny bit more complicated by leaving the original lockfile intact, but provide an alternative name for it via 'https://v3.yarnpkg.com/configuration/yarnrc#lockfileFilename'. That way we'll know we're doing the right thing even with that code path, since the base 'yarn.lock' case is covered by a unit test (alhough admittedly not the whole code path) and so not much can go wrong there really.